popsycle/synthetic.py

```diff
@@ -3146,3 +3154,454 @@ def calc_f(lambda_eff):
     return f
 
 
+def angdist(ra1, dec1, ra2, dec2):
+    '''
+    Takes two spherical coordinates and determines the angluar distance between
+    them.
+    Input:
+    ra1,dec1 [degrees] angular coordinates of the first point
+    ra2,dec2 [degrees] angular coordinates of the second point
+    Output:
+    distance [degrees] angular distance between the two points
+    '''
+    from numpy import pi, sin, cos, arccos
+    d2r = pi / 180.0
+    cosdist = (cos(np.absolute(ra1 - ra2) * d2r) * cos(np.absolute(dec1 * d2r)) * cos(np.absolute(dec2 * d2r)) +
+                sin(np.absolute(dec1 * d2r)) * sin(np.absolute(dec2 * d2r)))
+    distance = arccos(cosdist) / d2r
+    return distance
+
+
+def add_pbh(hdf5_file, ebf_file, output_root2, fdm=1, pbh_mass=40,
+            r_max=8.3, c=12.94, r_vir=200, inner_slope=.5, v_esc=550,
```

Comment:
This line of parameter inputs should be updated to reflect the new PBH halo sampling method.

Reply:
I unresolved it since it goes to the deeper discussion we are having about parameterization.

Related to our discussion above, we have defaults here like r_vir=200 which we don’t have a reference for. I think we are better off having defaults that from as few references as possible, both from a consistency stand point and ease of reporting in our paper. I know this is somewhat in conflict with my earlier conversation where I said it doesn’t really matter, where the argument was largely based on wanting to keep things easy. But given that I think the easiest path forward is now the better one we should go with that.